dropshot/Cargo.toml

```diff
@@ -54,7 +55,11 @@ version = "^0.10.2-dev"
 path = "../dropshot_endpoint"
 
 [dependencies.hyper]
-version = "0.14"
+version = "1"
```

Comment:
Same.

Reply:
I don't believe I used anything newer, but I can just stick it to 1.3. 🤷